images/installer/Dockerfile

```diff
@@ -1,51 +1,58 @@
-# Using playbook2image as a base
-# See https://github.com/openshift/playbook2image for details on the image
-# including documentation for the settings/env vars referenced below
-FROM registry.centos.org/openshift/playbook2image:latest
+FROM centos:7
 
 MAINTAINER OpenShift Team <dev@lists.openshift.redhat.com>
 
 LABEL name="openshift/origin-ansible" \
       summary="OpenShift's installation and configuration tool" \
       description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain and check an OpenShift cluster" \
-      url="https://github.com/openshift/openshift-ansible" \
-      io.k8s.display-name="openshift-ansible" \
-      io.k8s.description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain and check an OpenShift cluster" \
-      io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible"
+      url="https://github.com/openshift/openshift-ansible"
 
 USER root
 
+# install ansible and deps
+RUN yum install -y epel-release \
+ && yum-config-manager --disable epel \
+ && yum install -y --setopt=tsflags=nodocs --enablerepo=epel ansible \
+ && INSTALL_PKGS="python-lxml pyOpenSSL python2-cryptography skopeo openssl java-1.8.0-openjdk-headless httpd-tools" \
+ && yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS \
+ && rpm -V $INSTALL_PKGS \
+ && yum clean all
+
+ENV APP_ROOT=/opt/app-root \
+    USER_NAME=default \
+    USER_UID=1000 \
+    APP_HOME=/opt/app-root/src \
+    HOME=/opt/app-root/src \
+    PATH=$PATH:/opt/app-root/bin \
+#
+# The playbook to be run is specified via the PLAYBOOK_FILE env var.
+# This sets a default of openshift_facts.yml as it's an informative playbook
+# that can help test that everything is set properly (inventory, sshkeys)
+    PLAYBOOK_FILE=playbooks/byo/openshift_facts.yml \
+    OPTS="-v"
+
+COPY ./images/installer/bin /usr/bin
```

Comment:
I'm being picky, but this copies a README file into `/usr/bin`... I'd suggest moving the `README.md` inside `images/installer/bin` one level up and expand it a bit (see below)